molecule/default/vars/vars.yml

```diff
@@ -8,3 +8,4 @@ tas_single_node_cockpit:
   enabled: false
 tas_single_node_registry_username: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_USERNAME') }}"
 tas_single_node_registry_password: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_PASSWORD') }}"
+tas_single_node_user: testingUser
```

Comment:
Can we rename this variable? It makes it seem like it's part of the collection configuration and could confuse users who are looking for examples. I would suggest dropping the role prefix and also adding a code comment that this is used by molecule when setting up the VM.

Reply:
Renamed to testing_user + added comment